python/subunit/test_results.py

```diff
@@ -524,6 +532,16 @@ def addSuccess(self, test, details=None):
     def _failure_expected(self, test):
         return (test.id() in self._fixup_expected_failures)
 
+    def _apply_renames(self, test):
+        if not self._renames:
```

Comment:
Make this if is None after making the param None or func.

Reply:
Done.